server/services/startup.ts

```diff
@@ -29,11 +30,23 @@ api.registerService(new MeteorService());
 api.registerService(new NPSService(db));
 api.registerService(new RoomService(db));
 api.registerService(new SAUMonitorService());
-api.registerService(new VoipService(db));
-api.registerService(new OmnichannelVoipService(db));
 api.registerService(new TeamService(db));
 api.registerService(new UiKitCoreApp());
 
+let voipServiceInstance: VoipService;
```

Comment:
Can't this be handled at service level? Conditionally registering services based on settings is something i'm not in love with

Reply:
For sure, moved the logic to service level